lib/create-github-pr.js

```diff
@@ -1,8 +1,17 @@
-const octokit = require(`@octokit/rest`);
 const fixtureJsonStringify = require(`./fixture-json-stringify.js`);
 
 require(`./load-env-file.js`);
 
+const githubClient = new require(`@octokit/rest`)({
+  debug: false,
+  protocol: `https`,
```

Comment:
`protocol` and `host` were exactly the options that caused deprecation warnings. I suggest we go with the default values for all options. So just ``const githubClient = new require(`@octokit/rest`)();`` should be fine.

Reply:
Fixed.